frontend/app/src/components/helper/DataTable.vue

```diff
@@ -74,9 +72,12 @@ export default defineComponent({
 
     const scrollToTop = () => {
       const tableRef = get(table);
-      $vuetify.goTo(tableRef, {
-        container: document.querySelector('.app-main') as HTMLElement
-      });
+      if (tableRef) {
+        document.body.scrollTop =
+          tableRef.$el.getBoundingClientRect().top +
```

Comment:
I wonder if there is a vueuse util we could use here

Reply:
Updated